src/components/Card/Abilities/index.tsx

```diff
@@ -0,0 +1,30 @@
+import "./abilities.css";
+
+interface Props {
+  abilities: number[];
+}
+
+const abilityMap: { [key: number]: string } = {
+  0: "/assets/images/ability0.svg", 
+  1: "/assets/images/ability1.svg", 
+  2: "/assets/images/ability2.svg", 
+};
```

Comment:
This will have to be swapped out at some point, but it can definitely wait till the wrapper is slightly functional